packages/api-clients/open-api/bffApi.yml

```
@@ -2963,6 +2963,96 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /eservices/{eServiceId}/descriptors/{descriptorId}/instances:
```

Same as suggested in #1610, to apply also to catalogService etc.
```suggestion
  /templates/eservices/{eServiceId}/descriptors/{descriptorId}:
```